openedx/openedx-learning
# feat: Get drafts and get published functions [FC-0062]

Adds :

-  `get_drafts`: Given a list of publishable entities returns current drafts. Not all entities may have current drafts.
- `get_published functions`: Given a list of publishable entities returns those who have a published version.
    Not all entities may have current published version.

**Supporting information**

- Github issue: https://github.com/openedx/modular-learning/issues/234
- Internal link: https://tasks.opencraft.com/browse/FAL-3921
- Used in: https://github.com/openedx/edx-platform/pull/35734

**Testing instructions**

- Verify that tests cover all the new code
- Follow the testing instructions in: https://github.com/openedx/edx-platform/pull/35734
